features/security/README.md

```diff
@@ -29,9 +29,64 @@ This component has its own test suite.
 
 ### Strong typing
 
-Strong typing is ensured by [our strong typing functional test suite](strong_typing.md) and [the unit tests of the `AbstractItemNormalizer`
+Strong typing is ensured by [our "strong typing" functional test suite](strong_typing.feature) and [the unit tests of the `AbstractItemNormalizer`
 class](../../tests/Serializer/AbstractItemNormalizerTest.php).
 
-You might also be interested to see how extra attributes are ignored: [unknown_attributes.feature
+You might also be interested to see how extra attributes are ignored: [unknown_attributes.feature]
 
-To be continued.
+### Validate incoming content-types
```

Comment:
I remember these are supposed to be title case? :)

I don't think "content-types" makes much sense anyway, maybe just write it normally...